vitrina/settings.py

```diff
@@ -501,6 +501,14 @@
     },
 }
 
+# Celery settings
+CELERY_BROKER_URL = env("REDIS_URL", default="redis://127.0.0.1:6379/") + "3"
```

Comment:
Optional:
create variable and reuse together with already existing CACHES section
```suggestion
redis_url = env("REDIS_URL", default="redis://127.0.0.1:6379/")
CELERY_BROKER_URL = redis_url + "3"
```